docs/discussion/concepts/truncation.md

```diff
@@ -0,0 +1,130 @@
+# Truncation
+
+_Truncation_ occurs when the result of a unit conversion lands _in-between_ representable values in
+the destination type.  Along with [overflow], it's one of the two main [conversion risks] in units
+libraries.
+
+## Au's approach
+
+Au implements unit conversions as _sequences of operations_.  Some operations multiply or divide by
+a particular number.  Others cast a value from one type to another.
+
+Au checks every operation in the sequence to see whether any input values might cause truncation.
+If we find _any_ risk, we forbid the conversion --- at least, by default.  There may be valid
+reasons to perform such a conversion anyway.  If users are confident this is what they want, they
+can pass `ignore(TRUNCATION_RISK)` as a second argument to any conversion operator.
+
+Usually, even conversions with truncation risk will truncate for only some values, and not for
+others.  Users can check _individual values_ at runtime to see if that particular value is safe.
+`will_conversion_truncate(q, u)` examines converting the _specific_ `Quantity` value `q` to the new
+unit `u`.  If the conversion being checked also changes the representation type (the "rep"), to
+a new type `T`, `will_conversion_truncate<T>(q, u)` is the right form to use.
+
+These tools complement each other.  A common pattern is to check an individual value at runtime, and
+then call the actual conversion function with `ignore(TRUNCATION_RISK)` if the check passes.
+Remember, the `TRUNCATION_RISK` flag refers to the risk of the _conversion as a whole_.  If you've
+just proved that _your particular value_ is safe, then ignoring that _general_ risk is clearly fine.
+
+## Truncation in mathematical operations
+
+By "mathematical operations", we mainly mean multiplying or dividing by numerical constants.  To
+understand the truncation risk, we first put the rep into one of several categories, because the
+risk profile strongly depends on the category.
+
+The main categories are the arithmetic[^1] types: integral, and floating point.  We'll also briefly
+discuss how to think about other numeric types that we hope to support more fully in the future
+([#52]).
+
+[^1]: "Arithmetic" types are C++'s built-in numeric types: `int`, `double`, `uint64_t`, and so on.
+
+### Integral types
+
+Integral types are considered _exact_.  To see the implications for truncation, we need to consider
+various [types of conversion factors](../implementation/applying_magnitudes.md).
+
+First, we have multiplying by an integer.  This is easy: it stays within the domain of the integers,
+so it can never truncate.
+
+Next, we have multiplication by a non-integer rational (including reciprocal integers).  This will
```

Comment:
"First ... multiplying" and "Next ... multiplication" aren't consistent.  Not sure if they need to be, just pointing that out.